docs/config/services/deployments/production.js

```diff
@@ -1,16 +1,30 @@
 'use strict';
 
 var versionInfo = require('../../../../lib/versions/version-info');
-var cdnUrl = '//ajax.googleapis.com/ajax/libs/angularjs/' + versionInfo.cdnVersion;
+
+var googleCdnUrl = '//ajax.googleapis.com/ajax/libs/angularjs/';
+var angularCodeUrl = '//code.angularjs.org/';
+
+var cdnUrl = googleCdnUrl + versionInfo.cdnVersion;
+
+// The plnkr examples must use the code.angularjs.org repo for the snapshot,
+// and the cdn for the tagged version and, if the build is not tagged, the currentVersion.
+//
+// The currentVersion may not be available on the cdn (e.g. if built locally, or hasn't been pushed
+// yet). This will lead to a 404, but this is preferable to loading a version with which the example
+// might not work (possibly in subtle ways).
+var examplesCdnUrl = versionInfo.isSnapshhot ?
+  (angularCodeUrl + 'snapshot') :
+  (googleCdnUrl + (versionInfo.version || versionInfo.currentVersion));
```

Comment:
I thought about setting the cdnVersion. Snapshot is also possible, but my idea was that in the rare case of no .version + testing in plnkr (basically only local), it's better to have the immediate feedback of a 404 script instead of a (probably) working app, that might not be 100% compatible with the angular version it runs on.